test/support/factory.ex

```diff
@@ -113,4 +114,15 @@ defmodule Wanda.Factory do
       result: Map.get(attrs, :result, Enum.random([:passing, :warning, :critical]))
     }
   end
+
+  def execution_result_log_item_factory(attrs) do
```

Comment:
i find this names very hard to remember, to me this is an execution_result or a result. 
Consider splitting the factories if you have name clashing

Reply:
I see, there's different contexts involved.

The internal execution result computed by the engine and the execution result representation that gets appended to the log of execution results (reads _the one that gets saved in the database_)

However,
we can change `execution_result_log_item_factory` to `result_factory` if that sounds better to you, and keep it in the same factory.

`execution_result` would probably sound a bit clearer, unfortunately it would conflict with https://github.com/trento-project/wanda/blob/main/test/support/factory.ex#L69 and so would require splitting the factory.
For reference, something like this? https://hexdocs.pm/ex_machina/readme.html#where-to-put-your-factories

If you agree we can go lean and just rename `execution_result_log_item_factory` to `result_factory` at the moment.

Reply:
`result_factory` is ok to me

Reply:
changed to `result_factory`

```elixir
def result_factory(attrs) do
    ...
end

insert_list(3, :result)
```